ckcp/test.sh

```diff
@@ -6,10 +6,30 @@ set -euo pipefail
 # Uncomment the below line to enable debugging
 # set -x
 
-SCRIPT_DIR="$(cd "$(dirname "$0")" >/dev/null ; pwd)"
-KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
-export KUBECONFIG=$KUBECONFIG
-KUBECONFIG_KCP="${KUBECONFIG_KCP:-$SCRIPT_DIR/work/credentials/kubeconfig/kcp/ckcp-ckcp.default.pipeline-service-compute.kubeconfig}"
+usage() {
+
+    printf "Usage: WORK_DIR=/workspace CASES=pipelines ./test.sh\n\n"
+
+    # Parameters
+    printf "WORK_DIR: the location of the gitops files\n"
+    printf "CASES: the name of test casesm, the allowed cases are 'pipelines' or 'triggers' or 'pipelines,triggers'\n"
```

Comment:
`CASES: comma separated list of test cases. Test cases must be any of 'pipelines' or 'triggers'. All test cases are run by default.`

Reply:
fixed